components/nds-phase-banner/README.md

```diff
@@ -1,11 +1,97 @@
 # `@nice-digital/nds-phase-banner`
 
-> TODO: description
+> Phase banner component for the NICE Design System
+
+- [`@nice-digital/nds-phase-banner`](#nice-digitalphasebanner) - [Installation](#installation) - [Usage](#usage) - [React](#react) - [Props](#props) - [SCSS](#scss) - [HTML](#html)
+
+## Installation
+
+Install Node, and then:
+
+```sh
+npm i @nice-digital/nds-phase-banner --save
+```
 
 ## Usage
 
+### React
+
+Import the `PhaseBanner` component from the package and use within JSX:
+
+```jsx
+import React from "react";
+import { PhaseBaner } from "@nice-digital/nds-phase-banner";
+
+<PhaseBanner alpha>
+	This is a new service - your <a href='#'>feedback</a> will help us to improve it.
+</PhaseBanner>;
 ```
-const ndsPhaseBanner = require('@nice-digital/nds-phase-banner');
 
-// TODO: DEMONSTRATE API
+#### Props
```

Comment:
Is it worth also saying that any other props are spread over the wrapping element?

Reply:
I think we do this elsewhere in readmes